Client/qtTeamTalk/channel.ui

```diff
@@ -180,11 +180,22 @@
            </widget>
           </item>
           <item>
-           <widget class="QCheckBox" name="singletxchanBox">
-            <property name="text">
-             <string>No interruptions (no simultaneous voice transmission)</string>
-            </property>
-           </widget>
+           <layout class="QHBoxLayout" name="horizontalLayout_6">
+            <item>
+             <widget class="QCheckBox" name="singletxchanBox">
+              <property name="text">
+               <string>No interruptions (no simultaneous voice transmission)</string>
+              </property>
+             </widget>
+            </item>
+            <item>
+             <widget class="QToolButton" name="singletxButton">
+              <property name="text">
+               <string>...</string>
```

Comment:
We can maybe add an accessible text on this button to explicit for why it is, like on other buttons named "...".

Reply:
Yes, good idea. I've called it "Setup".